app/controllers/reservations_controller.rb

```diff
@@ -2,10 +2,18 @@
 
 class ReservationsController < ApplicationController
   before_action :set_screening, only: %i[new create]
+  before_action :set_reservation, only: %i[edit update destroy]
+
+  def index
+    @reservations = Reservation.includes(:screening, :tickets)
```

Comment:
you also refer to screenings hall and movie you need to include those too
```suggestion
    @reservations = Reservation.includes(:tickets, screening: { :movie, :hall } )
```